JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -209,12 +209,19 @@ public function denormalize($data, $class, $format = null, array $context = [])
         $instanceClass = $overrideClass ? get_class($context['object_to_populate']) : $class;
         $reflectionClass = new \ReflectionClass($instanceClass);
         if ($reflectionClass->isAbstract()) {
-            throw new InvalidArgumentException(
-                sprintf(
-                    'Cannot create an instance of %s from serialized data because it is an abstract resource',
-                    $instanceClass
-                )
-            );
+            $isAbstract = true;
+            if (is_subclass_of($resource->getEntityClass(), $instanceClass)) {
+                $instanceClass = $resource->getEntityClass();
```

Comment:
You have to check if the instanceClass is abstract or not and throw an exception if it is.